app/representers/api/musical_work_representer.rb

```diff
@@ -9,7 +9,7 @@ class Api::MusicalWorkRepresenter < Api::BaseRepresenter
   property :label
   property :album
   property :year
-  property :excerpt_length, as: :length
+  property :excerpt_length, as: :duration
```

Comment:
it seems consistency is sorta out the window on this one. I am fine with it as is.